app/views/job_invocations/_leapp_tab.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for(:javascripts) do %>
+    <%= webpacked_plugins_js_for :foreman_leapp %>
+  <% end %>
+  <% content_for(:stylesheets) do %>
+    <%= webpacked_plugins_css_for :foreman_leapp %>
+  <% end %>
+
+<%= react_component('JobInvocationLeappTab', { :job_invocation_id => @job_invocation.id })%>
```

Comment:
I think we usually have newline at the end, rubocop should flag this. Could you add those newlines to other files as well please?